addons/point_of_sale/static/tests/tours/helpers/ReceiptScreenTourMethods.js

```diff
@@ -107,3 +107,39 @@ export function shippingDateExists() {
         },
     ];
 }
+export function checkTaxDetails(tax, amount, base, total) {
+    return [
+        {
+            trigger: ".pos-receipt-taxes",
+            run: () => {
+                const tax_indexes = new Set();
+                const amount_indexes = new Set();
+                const base_indexes = new Set();
+                const total_indexes = new Set();
+
+                $(`.pos-receipt-taxes span:contains("${tax}")`).each(function () {
+                    tax_indexes.add($(".pos-receipt-taxes span").index(this));
+                });
+                $(`.pos-receipt-taxes span:contains("${amount}")`).each(function () {
+                    amount_indexes.add($(".pos-receipt-taxes span").index(this) - 1);
+                });
+                $(`.pos-receipt-taxes span:contains("${base}")`).each(function () {
+                    base_indexes.add($(".pos-receipt-taxes span").index(this) - 2);
+                });
+                $(`.pos-receipt-taxes span:contains("${total}")`).each(function () {
+                    total_indexes.add($(".pos-receipt-taxes span").index(this) - 3);
+                });
+
+                const commonElement = [...tax_indexes].find(
+                    (index) =>
+                        amount_indexes.has(index) &&
+                        base_indexes.has(index) &&
+                        total_indexes.has(index)
+                );
+                if (!commonElement) {
+                    throw new Error(`The tax details combination cannot be found on the receipt.`);
+                }
+            },
```

Comment:
```suggestion
trigger: `.pos-receipt-taxes span:contains('${tax}') ~ span:contains('${amount}') ~ span:contains('${base}') ~ span:contains('${total}')`,

```
![image](https://github.com/user-attachments/assets/6307a34f-9bbf-44c1-b00e-43428d65d6c9)

Reply:
Oh my! I made it more complicated than it needed to be :eyes: I didn't know `~` was a thing.